lib/utils/time-helpers.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include <QString>
+
+namespace advss {
+
+QString FormatRelativeTime(const float seconds);
```

Comment:
Sorry if I  just overlooked it, but why are you using a `float` here and not `double` / `uint64_t`?